docusaurus/docs/developer_guide/quickstart.md

```diff
@@ -43,7 +43,7 @@ Install the following dependencies:
 6. [Tilt](https://docs.tilt.dev/install.html) - k8s local development tool & environment manager
 
 :::note
-You might already have these installed if you've followed the [localnet instructions](../infrastructure/localnet.md).
+You might already have these installed if you've followed the [localnet instructions](./internal_infrastructure/localnet.md).
```

Comment:
I thought I would move our infra stuff (localnet and load-test) into a separate category to avoid confusion for community members.